.github/workflows/esp32-mkimage.yaml

```diff
@@ -31,13 +31,28 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
+  use-esp-idf:
```

Comment:
Can we add a short comment about the purpose of this step?

Reply:
done.